Production/govuk_ios/SwiftUIViews/GroupedList/GroupedList.swift

```diff
@@ -9,8 +9,20 @@ struct GroupedList: View {
             ZStack {
                 Color(backgroundColor ?? .clear)
                 VStack {
-                    ForEach(content, id: \.heading) { section in
-                        GroupedListSectionView(section: section)
+                    ForEach(content, id: \.heading?.title) { section in
+                        if section.heading?.icon != nil {
+                            GroupedListSectionIconView(
```

Comment:
Feels like you should still just be able to pass the section as a parameter to the views, rather than extracting each bit.  I've created a patch demonstrating this which I will slack to you.  I haven't run tests with it yet, so might need a little more work.  Let me know what you think.

Reply:
As discussed offline, agreed! Your diff passes tests straight out the box so no extra changes needed. Cheers